chips/cc26x2/src/i2c.rs

```diff
@@ -0,0 +1,247 @@
+//! I2C driver, cc26x2 family
+
+use core::cmp;
+use kernel::common::cells::{MapCell, OptionalCell};
+use kernel::common::registers::{ReadOnly, ReadWrite, WriteOnly};
+use kernel::common::StaticRef;
+use kernel::hil::i2c;
+
+#[repr(C)]
+struct I2CMasterRegisters {
+    /// Master slave address
+    msa: ReadWrite<u32, Address::Register>,
+    //mstat: ReadOnly<u32, Status::Register>,
```

Comment:
Should this be uncommented?

Reply:
Good catch! It shouldn't be uncommented, though. Turns out the MSTAT and MCTRL registers are the _same_ address---one is read one is write, and the bits have totally different meanings. So I need to figure out a reasonable way to represent that.